src/main/server/conf/jwt_marketplace_check.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+check_marketplace() {
+  INSTALL_DIRECTORY="/usr/local/antmedia"
+  REST_URL='http://localhost:5080/rest/v2/server-settings'
+
+  # Store original values
+  ORIGINAL_SERVER_JWT_CONTROL=$(grep "^server.jwtServerControlEnabled=" $INSTALL_DIRECTORY/conf/red5.properties)
```

Comment:
No need to store the original values and no need to change it temproraily. Just use them to call the REST API.